Mathlib/Combinatorics/SimpleGraph/Hasse.lean

```diff
@@ -121,4 +121,12 @@ theorem pathGraph_two_eq_top : pathGraph 2 = ⊤ := by
   ext u v
   fin_cases u <;> fin_cases v <;> simp [pathGraph, ← Fin.coe_covby_iff, Nat.covby_iff_succ_eq]
 
+/-- Hommorphism from smaller path graph to bigger path graph· -/
+protected def Hom.pathGraph {n m : ℕ} (hnm : n ≤ m) : pathGraph n →g pathGraph m where
```

Comment:
This should automatically generate your next lemma
```suggestion
@[simps]
protected def Hom.pathGraph {n m : ℕ} (hnm : n ≤ m) : pathGraph n →g pathGraph m where
```
You can check that it does by doing 
```suggestion
@[simps?]
protected def Hom.pathGraph {n m : ℕ} (hnm : n ≤ m) : pathGraph n →g pathGraph m where
```